conda_forge_tick/migration_runner.py

```diff
@@ -231,6 +231,7 @@ def run_migration_local(
 
     # it would be better if we don't re-instantiate ClonedFeedstockContext ourselves and let
     # FeedstockContext.reserve_clone_directory be the only way to create a ClonedFeedstockContext
+    # note: git_repo_owner is not set here because we cannot access it. Reading it possibly yields an incorrect value.
```

Comment:
Can you elaborate why this is really a problem (or not) and what would be the ideal situation?

Reply:
I changed the comment here: 75303763683773508362b83d8ab017fe95eb6233 Does that clear things up for you?
We should probably refactor this in a follow-up PR.